### PR TITLE
Add the FloatingActionButton

### LINKED ISSUE #17: Add FloatingActionButton
- Material Design Guideline
  - [Floating Action Button](https://material.io/design/components/buttons-floating-action-button.html)

### PR DESCRIPTION
**Pull request by yourina**
### Description of Change ###
This PR is for adding SnackBars of Material Design on Tizen

### Issues Resolved ### 
- fixes #17

### API Changes ###
 None

### Platforms Affected ### 
- Tizen

### Behavioral/Visual Changes ###
None

### Before/After Screenshots ### 
MFAB

### Testing Procedure ###
Use `FloatingActionButtonPage`

### PR Checklist ###

- [ ] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Review by rookiejava — COMMENTED** [review]
Does it working correctly for inherited properties and methods such as `Mode`. `Position`, `Opacity`, `MovementBlock` and so on?

**Comment by yourina** [comment]
@rookiejava I pushed the refactoring commit based on our off-line discussion.

File: src/Tizen.NET.MaterialComponents/Components/MConformant.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                WeightY = 1,
+                AlignmentX = -1,
+                AlignmentY = -1,
+            };
+            _layout.SetTheme("layout", Styles.MaterialApplication, "default");
+            _layout.Show();
+            base.SetContent(_layout);
+        }
+
+        public new void SetContent(EvasObject content)
```
Comment (rookiejava) [inline src/Tizen.NET.MaterialComponents/Components/MConformant.cs]: why don't you override `CreateHandle()` instead?

Comment (yourina) [reply src/Tizen.NET.MaterialComponents/Components/MConformant.cs]: Fixed it. Thank you.

File: src/Tizen.NET.MaterialComponents/Components/MConformant.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    _box.Show();
+                    _layout.SetPartContent(Parts.Layout.FloatingButton, _box);
+                }
+                return _box;
+            }
+        }
+
+        protected override IntPtr CreateHandle(EvasObject parent)
+        {
+            _layout = new Layout(parent)
```
Comment (rookiejava) [inline src/Tizen.NET.MaterialComponents/Components/MConformant.cs]: Handle = base.CreateHandle(parent);

**Review by rookiejava — DISMISSED** [review]

**Comment by rookiejava** [comment]
Could you update the `README.md` too?

File: README.md [hunk 3]
```diff
@@ -7,6 +7,9 @@ Material Components for Tizen .NET help developers execute [Material Design](htt
 - **MActivityIndicator**
   A circular progress indicator that supports determinate and indeterminate circular progress indicators. ([guideline](https://material.io/design/components/progress-indicators.html#circular-progress-indicators))
 
+- **MConformant**
+  A conformant allow the user to use a floating action button.
```
Comment (JoonghyunCho) [inline README.md]: You may be intended to mention like below?
"A material conformant allows ~~ " 
,or just "A conformant allows ~~"

Comment (yourina) [reply README.md]: @JoonghyunCho Fixed it. Thank you.

**Review by rookiejava — APPROVED** [review]

**Pull request merged**